app/src/main/java/com/gathering/android/auth/signup/repo/ApiSignUpRepository.kt

```diff
@@ -23,13 +24,11 @@ class ApiSignUpRepository @Inject constructor(
                     response: Response<GeneralApiResponse>
                 ) {
                     if (!response.isSuccessful) {
-                        onResponseReady(
-                            ResponseState.Failure(
-                                Exception(
-                                    RESPONSE_IS_NOT_SUCCESSFUL
-                                )
-                            )
-                        )
+                        if (response.code() == 409) {
```

Comment:
You can define a const for that like this: `private const val CONFLICT_HTTP_CODE = 409`

Reply:
done!